vm/ubpf_jit_x86_64.c

```diff
@@ -431,6 +431,14 @@ translate(struct ubpf_vm* vm, struct jit_state* state, char** errmsg)
             break;
 
         case EBPF_OP_LE:
+            if (inst.imm == 16) {
+                /* Truncate to 16 bits */
```

Comment:
Would you be willing to append to this comment (and the one below) a short note saying why we do not actually need to do any swap here?

Reply:
Done,